sql/mozfun/json/from_nested_map/udf.sql

```diff
@@ -0,0 +1,28 @@
+-- Definition for json.from_nested_map
+CREATE OR REPLACE function json.from_nested_map(input JSON)
+RETURNS json
+LANGUAGE js
+AS
+  """
+  if (input && Object.keys(input).length) {
+    for (const k in input) {
+      if (input[k] && input[k].length) {
```

Comment:
if `input` is an empty object this will return `null` correct? probably best to just return `input` at the bottom and never return `null`